client/src/app/site/motions/modules/motion-comment-section/components/motion-comment-section-list/motion-comment-section-list.component.html

```diff
@@ -13,52 +13,11 @@ <h2 translate>Comment fields</h2>
 </os-head-bar>
 
 <div class="head-spacer"></div>
-<mat-card *ngIf="commentSectionToCreate">
-    <mat-card-title translate>New comment field</mat-card-title>
-    <mat-card-content>
-        <form [formGroup]="createForm" (keydown)="keyDownFunction($event)">
-            <p>
-                <mat-form-field>
-                    <input formControlName="name" matInput placeholder="{{ 'Name' | translate }}" required />
-                    <mat-error *ngIf="!createForm.controls.name.valid">
-                        <span translate>Required</span>
-                    </mat-error>
-                </mat-form-field>
-            </p>
-            <p>
-                <os-search-value-selector
-                    ngDefaultControl
-                    [formControl]="this.createForm.get('read_groups_id')"
-                    [multiple]="true"
-                    listname="Groups with read permissions"
-                    [InputListValues]="this.groups"
-                ></os-search-value-selector>
-            </p>
-            <p>
-                <os-search-value-selector
-                    ngDefaultControl
-                    [formControl]="this.createForm.get('write_groups_id')"
-                    [multiple]="true"
-                    listname="Groups with write permissions"
-                    [InputListValues]="this.groups"
-                ></os-search-value-selector>
-            </p>
-        </form>
-    </mat-card-content>
-    <mat-card-actions>
-        <button [disabled]="createForm.invalid" mat-button (click)="create()">
-            <span translate>Create</span>
-        </button>
-        <button mat-button (click)="commentSectionToCreate = null">
-            <span translate>Cancel</span>
-        </button>
-    </mat-card-actions>
-</mat-card>
 <mat-accordion class="os-card">
     <mat-expansion-panel
         *ngFor="let section of this.commentSections"
         (opened)="openId = section.id"
-        (closed)="panelClosed(section)"
+        (closed)="openId = null"
```

Comment:
`opened` on `closed` can probably be ignored, since the default case of mat-accordion `multi=false` should work here